easybuild/easyconfigs/d/Datasets/Datasets-2.16.1-foss-2022a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = "PythonBundle"
+
+name = 'Datasets'
+version = '2.16.1'
+
+homepage = 'https://github.com/huggingface/datasets'
+description = """
+The largest hub of ready-to-use datasets for ML models with fast, easy-to-use and efficient
+data manipulation tools.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('aiohttp', '3.8.3'),
+    ('dill', '0.3.6'),
+    ('Arrow', '8.0.0'),
+    ('PyYAML', '6.0'),
+    ('tqdm', '4.64.0'),
+    ('python-xxhash', '3.1.0'),
+    ('pyarrow-hotfix', '0.6'),
```

Comment:
Instead of adding a pyarrow-hotfix and patching that I'd probably consider adding pyarrow>=14 directly and patching out the requirement for hotfix from datasets instead https://github.com/huggingface/datasets/blob/main/setup.py#L116-L119

Reply:
There is already a Arrow/8.0.0 in foss/2022a so you can disregard this comment.